app/javascript/app/components/ndc/ndc-component.jsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
+import Proptypes from 'prop-types';
 import Header from 'components/header';
 import Map from 'components/map';
 
-import { feature } from 'topojson-client';
-import geoData from 'app/data/world-50m.json';
+import paths from 'app/data/world-50m-paths';
 
-const paths = feature(geoData, geoData.objects[Object.keys(geoData.objects)[0]])
-  .features;
+const NDC = (props) => {
+  const handleCountryClick = (geography) => {
+    props.history.push(`ndcs/${geography.id}`);
```

Comment:
But I need the history prop that the `withRouter` HOC provide me in the component, so could we do it in an action?